apis/apps/v1alpha1/defaults.go

```diff
@@ -55,7 +55,7 @@ func setDefaultSidecarContainer(sidecarContainer *SidecarContainer) {
 
 func setSidecarSetUpdateStratety(strategy *SidecarSetUpdateStrategy) {
 	if strategy.Type == "" {
-		strategy.Type = NotUpdateSidecarSetStrategyType
+		strategy.Type = RollingUpdateSidecarSetStrategyType
```

Comment:
the change in default values should be highlighted in release notes

Reply:
it seems that the default value is RollingUpdateSidecarSetStrategyType in previous release